src/components/Simulation/index.js

```diff
@@ -84,4 +200,4 @@ Simulation.propTypes = {
   }),
 };
 
-export default Simulation;
+export default withTranslation("translations")(Simulation);
```

Comment:
Since this is a functional component, no reason to not use the translation hook I guess